cartesian_velocity_controller:
    type: franka_reactive_controller/CartesianVelocityNodeController
    arm_id: panda
    # If no new velocity commands are received in time (seconds)
    # Then set 0 velocity.
    max_duration_between_commands: 0.05
    rate_limiting:
        linear:
            velocity: 0.01
            acceleration: 2.0
            jerk: 500.0
        angular:
            velocity: 0.5
            acceleration: 2.0
            jerk: 500.0
    stop_on_contact: true

cartesian_pose_node_controller:
    type: franka_reactive_controller/CartesianPoseNodeController
    arm_id: panda
    # If no new velocity commands are received in time (seconds)
    # Then set 0 velocity.
    max_duration_between_commands: 0.025
    rate_limiting:
        linear:
            velocity: 0.05
            acceleration: 1.0
            jerk: 500.0
        angular:
            velocity: 0.5
            acceleration: 2.0
            jerk: 500.0
    stop_on_contact: true
